README.md

```diff
@@ -11,6 +11,24 @@ For further information:
 - Fast Downward main repository: <https://github.com/aibasel/downward>
 
 
+## Tested software versions
+
+This version of Fast Downward has been tested with the following systems,
+compilers and tools:
+
```

Comment:
The comment below is not telling me which combinations are tested, especially, this looks as if we would already test Windows and MacOS with both LP solvers. 
I would at least at a comment about the supported LP solvers, e.g. add to Ubuntu 'with CPLEX and SoPlex' (I will do that for Windows.